### PR TITLE
Added StartupWithIStartupFilter

### PR DESCRIPTION
**Pull request by jhkimnew**
Added StartupWithIStartupFilter class in order to apply it only for the test that requires disabling the graceful shutdown feature, not for the default test case.

### CODE REVIEW

File: test/AspNetCoreModule.Test/FunctionalTestHelper.cs [hunk 1]
```diff
@@ -718,6 +718,7 @@ public enum ReturnValueType
                     if (!isGraceFullShutdownEnabled)
                     {
                         iisConfig.SetANCMConfig(testSite.SiteName, testSite.AspNetCoreApp.Name, "environmentVariable", new string[] { "GracefulShutdown", "disabled" });
+                        iisConfig.SetANCMConfig(testSite.SiteName, testSite.AspNetCoreApp.Name, "environmentVariable", new string[] { "ANCMTestStartupClassName", "StartupWithIStartupFilter" });
```
Comment (Tratcher) [inline test/AspNetCoreModule.Test/FunctionalTestHelper.cs]: StartupWithShutdownDisabled?

Comment (jhkimnew) [reply test/AspNetCoreModule.Test/FunctionalTestHelper.cs]: I agree. I renamed the class name.

File: test/AspNetCoreModule.TestSites.Standard/Program.cs [hunk 2]
```diff
@@ -98,6 +93,18 @@ public static void Main(string[] args)
                     })
                     .UseConfiguration(config)
                     .UseStartup<Startup>();
+                }
+                else
+                {
+                    throw new System.Exception("Invalid startup class name : " + startUpClassString);
```
Comment (jkotalik) [inline test/AspNetCoreModule.TestSites.Standard/Program.cs]: Nit "throw new Exception();

**Review by jkotalik — APPROVED** [review]
Small nit but looks good to me

**Pull request merged**